js/combobox.js

```diff
@@ -270,10 +279,13 @@
 		},
 
 		inputchanged: function (e, extra) {
+			var val = $(e.target).val();
 			// skip processing for internally-generated synthetic event
 			// to avoid double processing
-			if (extra && extra.synthetic) return;
-			var val = $(e.target).val();
+			if (extra && extra.synthetic) {
+				this.selectByText(val);
```

Comment:
If the synthetic event of the menu click happens. The menu item never gets the `.selected` class.